src/formatDate.js

```diff
@@ -50,7 +50,50 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const YEAR_FORMAT_4 = 'YYYY';
+  const YEAR_FORMAT_2 = 'YY';
+  const MONTH_FORMAT = 'MM';
+  const DAY_FORMAT = 'DD';
+
+  const yearIndexOld = fromFormat.indexOf(YEAR_FORMAT_4) >= 0
+    ? fromFormat.indexOf(YEAR_FORMAT_4) : fromFormat.indexOf(YEAR_FORMAT_2);
+  const dayIndexOld = fromFormat.indexOf(DAY_FORMAT);
+  const monthIndexOld = fromFormat.indexOf(MONTH_FORMAT);
+
+  const yearIndexNew = toFormat.indexOf(YEAR_FORMAT_4) >= 0
```

Comment:
Similarly, repeated logic for getting the year index. Consider creating a helper function to avoid code duplication